personalized_nlp/datasets/emotions_perspective/emotions_perspectives.py

```diff
@@ -0,0 +1,80 @@
+from lib2to3.pytree import Base
+from typing import List
+
+import pandas as pd
+import os
+
+from personalized_nlp.settings import STORAGE_DIR
+from personalized_nlp.utils.data_splitting import split_texts
+from personalized_nlp.datasets.datamodule_base import BaseDataModule
+
+class EmotionsPerspectiveDataModule(BaseDataModule):
+    def __init__(
+            self, 
+            language: str = 'english',
```

Comment:
`language` field is not necessarilly needed because all texts are in English now so we can remove it.